cli/cmd/cluster.go

```diff
@@ -17,28 +17,33 @@ limitations under the License.
 package cmd
 
 import (
+	"encoding/base64"
 	"fmt"
 	"os"
 	"path/filepath"
 	"regexp"
 	"strings"
 	"time"
 
+	"github.com/aws/aws-sdk-go/aws"
 	"github.com/aws/aws-sdk-go/service/autoscaling"
 	"github.com/aws/aws-sdk-go/service/ec2"
+	"github.com/aws/aws-sdk-go/service/eks"
 	"github.com/aws/aws-sdk-go/service/elbv2"
 	"github.com/aws/aws-sdk-go/service/s3"
 	"github.com/cortexlabs/cortex/cli/cluster"
 	"github.com/cortexlabs/cortex/cli/types/cliconfig"
 	"github.com/cortexlabs/cortex/cli/types/flags"
 	"github.com/cortexlabs/cortex/pkg/consts"
-	"github.com/cortexlabs/cortex/pkg/lib/aws"
+	"github.com/cortexlabs/cortex/pkg/health"
+	awslib "github.com/cortexlabs/cortex/pkg/lib/aws"
```

Comment:
there are other usages of the same package with the alias `awslib`. I just kept it consistent.

I prefer the `lib` as a suffix because it's more intuitive to type with the IDE autocomplete

Reply:
:+1: I don't have a strong preference either way, keeping it consistent would be good (across all of our libs)